cg/server/api.py

```diff
@@ -149,40 +149,56 @@ def parse_cases():
     return jsonify(cases=cases, total=len(cases))
 
 
+def _get_current_customers() -> Optional[List[Customer]]:
+    """Return customers if the current user is not an admin."""
+    if not g.current_user.is_admin:
+        return g.current_user.customers
+    return None
+
+
+def _get_cases(
+    status: str, enquiry: Optional[str], action: Optional[str], customers: Optional[List[Customer]]
+) -> List[Family]:
+    """Get cases based on the provided filters."""
+    if status == "analysis":
+        return db.cases_to_analyze(pipeline=Pipeline.MIP_DNA)
```

Comment:
Hmm, this one should take a pipeline arg and not be hard coded to MIP_DNA. Perhaps chsnge in a future PR (good first issue)

Reply:
https://github.com/Clinical-Genomics/cg/issues/1889